File: app/src/at-myhuynh/java/asiantech/internship/summer/ex04/PrimeFactor.java
package asiantech.internship.summer.ex04;

import java.util.List;

import asiantech.internship.summer.common.Common;

public class PrimeFactor {

	public static void main(String[] args) {
		int number = 28;
		List<String> listPrime = Common.primeFactor(number);
		System.out.println(String.join("x", listPrime));
	}
}

File: app/src/at-myhuynh/java/asiantech/internship/summer/kotlin/ex04/PrimeFactor.kt
package asiantech.internship.summer.kotlin.ex04

import asiantech.internship.summer.kotlin.common.Common

object PrimeFactor {

    @JvmStatic
    fun main(args: Array<String>) {
        val number = Common.input("Nhập vào n: ")
        val listPrime = Common.primeFactor(number)
        println(listPrime.joinToString(
            separator = "x",
            prefix = "",
            postfix = "",
            limit = listPrime.size,
            truncated = ""
        ))
    }
}
